drivers/clock_control/clock_stm32l0_l1.c

```diff
@@ -26,29 +25,59 @@
 #define pll_div(v) z_pll_div(v)
 
 /**
- * @brief Set up pll configuration
+ * @brief Return PLL source
  */
-int config_pll_sysclock(void)
+__unused
+static uint32_t get_pll_source(void)
 {
-	uint32_t pll_source, pll_mul, pll_div;
-
-	pll_mul = pll_mul(STM32_PLL_MULTIPLIER);
-	pll_div = pll_div(STM32_PLL_DIVISOR);
-
 	/* Configure PLL source */
 	if (IS_ENABLED(STM32_PLL_SRC_HSI)) {
-		pll_source = LL_RCC_PLLSOURCE_HSI;
+		return LL_RCC_PLLSOURCE_HSI;
 	} else if (IS_ENABLED(STM32_PLL_SRC_HSE)) {
-		pll_source = LL_RCC_PLLSOURCE_HSE;
-	} else {
+		return LL_RCC_PLLSOURCE_HSE;
+	}
+
+	return -ENOTSUP;
+}
+
+/**
+ * @brief Set up pll configuration
+ */
+__unused
+int config_pll_sysclock(void)
+{
+	uint32_t pll_source;
+
+	pll_source = get_pll_source();
+	if (pll_source < 0) {
 		return -ENOTSUP;
 	}
 
-	LL_RCC_PLL_ConfigDomain_SYS(pll_source, pll_mul, pll_div);
+	LL_RCC_PLL_ConfigDomain_SYS(pll_source,
+				    pll_mul(STM32_PLL_MULTIPLIER),
+				    pll_div(STM32_PLL_DIVISOR));
 
 	return 0;
 }
 
+/**
+ * @brief Return pllout frequency
+ */
+__unused
+uint32_t get_pllout_frequency(void)
+{
+	uint32_t pll_source;
+
+	pll_source = get_pll_source();
+	if (pll_source < 0) {
```

Comment:
`get_pll_source()` returns `uint32_t`.
Therefore, `if (pll_source < 0)` is always `false`.